.github/workflows/codeql-analysis.yml

```
@@ -23,7 +23,7 @@ jobs:
       - name: Setup Python3
         uses: actions/setup-python@v4
         with:
-          python-version: '3.8.10'
+          python-version: '3.9.16'
```

```suggestion
          python-version: '3.9.17'
```

nit since you also updated the version to 3.9.17

---

Unfortunately the setup-python action isn't yet able to install 3.9.17 (at least it was not as of yesterday, I can check again today).

---

They will add support for 3.9.17 in this PR: https://github.com/actions/python-versions/pull/232/files (still open as of writing this comment)

---

Ok let's keep it as it then!